src/scripts/components/Tooltip.jsx

```diff
@@ -206,6 +233,22 @@ class Tooltip extends React.Component {
    */
   removeTooltip(tooltip) {
     tooltip.remove();
+    this.tooltipId = undefined;
+  }
+
+  /**
+   * Open or close tooltip based on its current state.
+   * Used only on touch devices that rely on click events since a click could
+   * call for either opening or closing.
+   * @param {event} e
+   * @return {void}
+   */
+  toggleTooltip(e) {
+    if (!this.isTooltipOpen) {
```

Comment:
Personal preference, so leave this as is if you want. Since you're doing an "either or" here and your variable is named `isTooltipOpen`, I find it easier to read like this (notice there is no negation to read):
```
if (this.isTooltipOpen) {
  this.closeTooltip(e);
} else {
  this.createTooltip(e);
}
```